### PR TITLE
Update docs for light style box

### PR DESCRIPTION
**Pull request by hcwong**
**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Documentation update

Closes https://github.com/MarkBind/vue-strap/pull/119/files by updating documentation.

**What is the rationale for this request?**
Update documentation as boxes have been enhanced

**What changes did you make? (Give an overview)**
Updated props and gave an example on the docs

**Proposed commit message: (wrap lines at 72 characters)**

Update docs for light style TipBox

### CODE REVIEW

File: docs/userGuide/syntax/boxes.mbdf [hunk 1]
```diff
@@ -101,6 +104,9 @@
 <box type="info" icon=":rocket:">
     info, with rocket icon
 </box>
+<box type="definition" light>
+    light definition
+</box>
 </span>
 </include>
```
Comment (damithc) [inline docs/userGuide/syntax/boxes.mbdf]: We may want to show the full set of box examples once more in the light mode i.e., one set in default styles and one set in light styles

File: docs/userGuide/syntax/boxes.mbdf [hunk 2]
```diff
@@ -78,6 +78,60 @@
...
+<box type="tip" light>
+    tip light
+</box>
+<box type="definition" light>
+    definition light
+</box>
+</span>
+</include>
 
 **You can customize the TipBox's appearance.**
```
Comment (damithc) [inline docs/userGuide/syntax/boxes.mbdf]: As long as we are touching this code, can we also change this `TipBox` -> `box`?

File: docs/userGuide/syntax/boxes.mbdf [hunk 3]
```diff
@@ -101,6 +155,9 @@
 <box type="info" icon=":rocket:">
     info, with rocket icon
 </box>
+<box type="definition" light>
+    light definition
+</box>
```
Comment (yamgent) [inline docs/userGuide/syntax/boxes.mbdf]: Since this is already displayed above, this code doesn't seem to be necessary anymore?

Comment (hcwong) [reply docs/userGuide/syntax/boxes.mbdf]: Ah sorry forgot to push latest commit. will fix

**Pull request merged**